### PR TITLE
feature: TR-857 RTL title bar PoC

### PR DESCRIPTION
**Pull request by lecosson**
**Related to:** https://oat-sa.atlassian.net/browse/TR-857

**Changes:**
- checking list of RTL locales and adding according attribute to implement necessary styles
- using implemented util/locale helpers
- update package-lock to use necessary tao-test-runner-qti-fe branch
- styles bundled

**How to check:**
- please check description of [mentioned PR](https://github.com/oat-sa/tao-test-runner-qti-fe/pull/390)

**Note:**
 - [ ] package version must be replaced after release

**Requires:**
 - [ ] [tao-test-runner-qti-fe PR](https://github.com/oat-sa/tao-test-runner-qti-fe/pull/390)
 - [ ] [tao-core PR](https://github.com/oat-sa/tao-core/pull/2928)
 - [ ] [tao-core-sdk-fe PR](https://github.com/oat-sa/tao-core-sdk-fe/pull/109)

### CODE REVIEW

File: views/js/controller/runner/runner.js [hunk 1]
```diff
@@ -89,6 +91,9 @@ define([
                 plugins : config && config.providers && Object.keys(config.providers.plugins)
             });
 
+            const rtl = [ ... locale.getConfig().rtl || [] ].includes(context.locale);
+            $('.delivery-scope').toggleClass('rtl', rtl);
```
Comment (jsconan) [inline views/js/controller/runner/runner.js:95]: **nitpick:** The result is there, and we have the RTL flag.
However:
- The RTL detection should be done by a helper inside `util/locale`. In the current state, the code is relying on internal structure that may change for any reason. It is better to rely on a public contract. For instance, we could have a `isLanguageRTL(lang)` helper or a more generic `getLanguageDirection(lang)` in `util/locale`.
- It is enough to set a CSS class to switch the direction? Maybe this also needs the `dir` attribute to be set as well

**Review by jsconan — COMMENTED** [review]
The genral idea is good, but I have some comments on the implementation. Please see my comments

Edit: please also rebase the PR to `develop` instead `master`

File: views/js/controller/runner/runner.js [hunk 2]
```diff
@@ -89,6 +91,8 @@ define([
                 plugins : config && config.providers && Object.keys(config.providers.plugins)
             });
 
+            $('.delivery-scope').attr({dir: locale.getLanguageDirection(context.locale)});
```
Comment (jsconan) [inline views/js/controller/runner/runner.js:94]: **thought:** Looks good if we consider not only the test runner but the whole page.
However, I'd rather place this aside with the loading bar start for better consistency, [around line 175](https://github.com/oat-sa/extension-tao-testqti/pull/2020/files#diff-9c3d580247a181bfe1ce77bb1052b6e90941ac5231603179fb7bea049cc0df8fR175)

**Review by jsconan — COMMENTED** [review]
Looks good.
Just one remark on the place for the change

**Review by jsconan — APPROVED** [review]
Looks good for me.
Note: this also impact the whole layout, other stories will need to be implemented, like the one on the item runner to apply the language settings to the content. For the time being the whole content is set RTL.

Remark: the type of commit is not correct for the last one. It says `hotfix:` but it should rather be `fix:`.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by btamas — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**